<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelemetryMerger</class>
 <widget class="QMainWindow" name="TelemetryMerger">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>516</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Telemetry Merger</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="leftPane">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="toolbar">
        <item>
         <widget class="QToolButton" name="itemUp_ToolButton">
          <property name="toolTip">
           <string>Move source up</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset theme="go-up">
            <normaloff>.</normaloff>.</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="itemDown_ToolButton">
          <property name="toolTip">
           <string>Move source down</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset theme="go-down">
            <normaloff>.</normaloff>.</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="sources_TableView"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QListView" name="listView"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>937</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
